various-receipes/authentication/authentication-middleware.js

```diff
@@ -0,0 +1,26 @@
+const jwt = require("jsonwebtoken");
+const config = require("./config");
+
+const authenticationMiddleware = (req, res, next) => {
+    //This is where some authentication & authorization logic comes, for example a place to
+    //validate JWT integrity
+    let succeeded = false;
+    const authenticationHeader = req.headers["authorization"];
+    console.log(`About to authorize a request, header is ${authenticationHeader}`);
```

Comment:
Is JWT public?
If not, we shouldn't log it (nor encourage our readers to hehe)

Reply:
Good point, changing